lib/src/ui/movie_list.dart

```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Popular Movies'),
+      ),
+      body: StreamBuilder(
+        stream: movieBloc.allMovies,
+        builder: (context, AsyncSnapshot<Movie> snapshot) {
+          if (snapshot.hasData) {
+            return buildList(snapshot.data);
+          } else if (snapshot.hasError) {
+            return Text(snapshot.error.toString());
+          }
+          return Center(child: CircularProgressIndicator());
+        },
+      ),
+    );
+  }
+
+  Widget buildList(Movie? data) {
+    return GridView.builder(
+        itemCount: data?.results.length,
+        gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+            crossAxisCount: NumericConstants.crossAxisCount),
+        itemBuilder: (BuildContext context, int index) {
+          return GridTile(
+            child: Container(
+              margin:
```

Comment:
ctrl + alt + L should fix this identation

Reply:
fixed